zebra/zebra_netns_notify.c

```diff
@@ -92,7 +92,11 @@ static void zebra_ns_notify_create_context_from_entry_name(const char *name)
 		zlog_warn("NS notify : failed to create VRF %s", name);
 		return;
 	}
+	if (zserv_privs.change(ZPRIVS_RAISE))
+		zlog_err("Can't raise privileges");
 	ret = vrf_netns_handler_create(NULL, vrf, netnspath, ns_id);
```

Comment:
why do we sometimes need to raise priviledges to call this function and sometimes we do not?

Reply:
now, on every place where vrf_netns_handler_create() is called, we try to raise the privilege level.

Reply:
There is no reason to raise privileges sometimes and other times not.
this is why I make that patch.